concent_api/core/tests/test_integration_force_payment.py

```diff
@@ -231,65 +231,6 @@ def test_that_if_provider_sends_force_payment_with_two_different_provider_ethere
             }
         )
 
-    def test_provider_send_force_payment_beyond_payment_time_concent_should_reject(self):
```

Comment:
this was removed because now GM don't allow to create such messages with wrong `payment_ts` or what was the reason?

Reply:
That's right, GM now does not allow to create messages with `payment_ts` from future